File: sir/API/src/main/java/fr/istic/sir/doodle/form/InscriptionForm.java
package fr.istic.sir.doodle.form;

import java.util.List;

import fr.istic.sir.doodle.entities.Allergie;
import fr.istic.sir.doodle.entities.Preference;
// import fr.istic.sir.doodle.entities.User;

public class InscriptionForm {
	
	public UserDTO getUser() {
		return user;
	}
	public void setUser(UserDTO user) {
		this.user = user;
	}
	public List<Allergie> getAllergies() {
		return allergies;
	}
	public void setAllergies(List<Allergie> allergies) {
		this.allergies = allergies;
	}
	public List<Preference> getPreferences() {
		return preferences;
	}
	public void setPreferences(List<Preference> preferences) {
		this.preferences = preferences;
	}
	
	private UserDTO user;
	private List<Allergie>allergies;
	private List<Preference>preferences;
	public InscriptionForm() {	}	

}

File: sir/WEB/src/app/register/InscriptionForm.ts
import { User } from '../service/authentication.service';
import { Allergie } from './Allergie';
import { Preference } from './Preferences';

export class InscriptionForm {
  public user: User;
  public allergies: Allergie[];
  public preferences: Preference[];

  constructor(user:User,allergies:Allergie[], preferences:Preference[]) {
       this.user = user;
       this.allergies = allergies;
       this.preferences = preferences;
  }

}


